### ISSUE TITLE
Feature Request: Landscape brushes to paint mesh mountains and other things we may like

### ISSUE THREAD

**Issue by delebash** [GameDev4K/unreal-open-land#28]
This would give use an easy way to add predefined yet customizable things to our landscape.

**Comment by arunoda** [comment]
Do you mean Alpha brushes or Landmass brushes?

**Comment by delebash** [comment]
The same concept that brushify uses so users can quickly create a scene.

**Comment by arunoda** [comment]
Okay. They use an Alpha brush. Basically, you can use them with OpenLand as well.
We don't have any alpha brushes (basically heightmaps).
So, your idea is to ship some of those?

Is it?

**Comment by delebash** [comment]
Yes, just a nice to have. I think it should be lower on the priority list but a plus overall.
